src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball(String colour, int number) {
```

Comment:
colour -> color. Here's a cool shortcut to replacing some string with another string everywhere in the project: https://www.jetbrains.com/help/idea/finding-and-replacing-text-in-project.html#replace_search_string_in_project